deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -681,6 +681,17 @@ def check_feed_uncompressed_successfully(wazuh_log_monitor, feed, update_positio
     )
 
 
+def modify_metadata_vuldet_feed(feed, timestamp):
+    """Function to modify the timestamp value of the metadata table for a specific feed.
+
+    Args:
+        feed (str): The feed name.
+        timestamp (str): The new timestamp value to set.
+    """
+    query_string = f'update METADATA set TIMESTAMP="{timestamp}" where TARGET="{feed}"'
```

Comment:
Be careful with this, for now whenever you make a query to the database you have to stop `wazuh-db` and start it afterwards.

As a future improvement, we have to delegate this to the make_query function, but for now, you have to do it like this (as it is done in the rest of vuln detector tests)

Reply:
Fixed here: 8c08c1be28917fd543839d17dabc0dc722406080